.tests/technitium-logs/technitium.log

```diff
@@ -0,0 +1 @@
+[2025-07-28 05:34:01 UTC] [169.254.44.3:50051] DnsServerCore.DnsWebServiceException: Invalid username or password for user: admin
```

Comment:
Sorry didnt notice this before, we are trying to remove WAN ips from our tests can you replace `169.254.44.3` with an internal like `192.168.1.1` and re run the hubtests.

apologies before for not noticing this.